src/views/connected/admin/gestionnaires/Gestionnaire.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import dayjs from 'dayjs';
+import { gestionnaireActions, alerteEtSpinnerActions } from '../../../../actions';
+import FormSuppressionGestionnaire from './FormSuppressionGestionnaire';
+
+function Gestionnaire({ gestionnaire }) {
+  const dispatch = useDispatch();
+  const [confirmSuppressionGestionnaire, setConfirmSuppressionGestionnaire] = useState(false);
+
+  const resendInvitGestionnaire = () => {
+    window.scrollTo(0, 0);
```

Comment:
Y a une fonction normalement dans les utils

```suggestion
    scrollTopWindow();
```

Reply:
modifié